qa/tasks/cephfs/test_volume_client.py

```diff
@@ -1089,6 +1089,448 @@ def test_recover_metadata(self):
             auth_id=guestclient["auth_id"],
         )))
 
+    def test_update_old_style_auth_metadata_to_new_during_recover(self):
+        """
+        The older versions of CephVolumeClient stores the
+        subvolume data in auth metadata file with 'volumes'
+        key as there was no subvolume namespace. It doesn't
```

Comment:
Can you not say that from nautilus onwards 'volumes' created by ceph_volume_client were renamed and used as CephFS subvolumes accessed via the ceph-mgr interface? Not sure what you mean by subvolume namespace?

Reply:
I meant the subvolume abstraction was not there before and is brought in by mgr/volumes. I can re-phrase, no issues.

Reply:
Done